Samples/WidgetAdvSample/Package.appxmanifest

```diff
@@ -142,6 +142,7 @@
         <Interface Name="Microsoft.Gaming.XboxGameBar.Private.IXboxGameBarWidgetActivatedEventArgsPrivate" InterfaceId="782535A7-9407-4572-BFCB-316B4086F102" />
         <Interface Name="Microsoft.Gaming.XboxGameBar.Private.IXboxGameBarWidgetHost" InterfaceId="5D12BC93-212B-4B9F-9091-76B73BF56525" />
```

Comment:
Do we need to add IXboxGameBarWidgetHost2?

Reply:
Added to all files